internal/flags.go

```diff
@@ -37,6 +37,8 @@ const (
 	flagLogFileSize     = "logFileSize"
 	flagLogFileCount    = "logFileCount"
 	flagLogFileMaxAge   = "logFileMaxAge"
+	flagCert            = "cert"
```

Comment:
Is it correct to use such generic names? How are we going to add MQTTS later on? Overall concept has to be integrated. For example, these flags can be: downloadCert, downloadKey.